sdk/python/feast/request_feature_view.py

```diff
@@ -40,7 +40,7 @@ class RequestFeatureView(BaseFeatureView):
     def __init__(
         self,
         name: str,
-        request_data_source: RequestDataSource,
+        request_data_source: RequestSource,
```

Comment:
change field name? also, change to use kwargs? Or is that a different PR?

Reply:
This is a different PR, I am not deprecating feature view params here.